Vagrantfile

```diff
@@ -11,24 +11,33 @@ $memory = ENV.fetch("ISLANDORA_VAGRANT_MEMORY", "4096")
 $hostname = ENV.fetch("ISLANDORA_VAGRANT_HOSTNAME", "islandora8")
 $virtualBoxDescription = ENV.fetch("ISLANDORA_VAGRANT_VIRTUALBOXDESCRIPTION", "Islandora 8")
 
-# The Vagrant Base Box to use. Currently works with 'ubuntu/focal64'.
-$vagrantBox = ENV.fetch("ISLANDORA_DISTRO", "ubuntu/focal64")
```

Comment:
`$vagrantBox` is called later on line 65 in your version, but I think the whole `if` clause there is now irrelevant.

Reply:
I was actually thinking maybe I should re-add this as I didn't remove any of the Redhat stuff internally. So I'm not sure why I decided to force it to Ubuntu.